src/test/groovy/jp/co/soramitsu/devops/tasks/CustomFilesTaskTest.groovy

```diff
@@ -0,0 +1,44 @@
+package jp.co.soramitsu.devops.tasks
+
+import jp.co.soramitsu.devops.SoraTask
+import jp.co.soramitsu.devops.utils.GradleProjectExecutor
+import jp.co.soramitsu.devops.utils.TestUtils
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class CustomFilesTaskTest extends Specification {
+
+    @Unroll
+    def "[#projectName] app has custom file copy done"() {
+        given: "gradle project"
+        def result
+        def project = new GradleProjectExecutor(new File("./projects/${projectName}"))
```

Comment:
Look at this test.
https://sourcegraph.com/github.com/soramitsu/gradle-sora-plugin/-/blob/src/test/groovy/jp/co/soramitsu/devops/misc/ExtensionTest.groovy#L13-14

Pass here a temporary dir